pkgs/tools/package-management/nix/default.nix

```diff
@@ -38,59 +38,10 @@ let
         aws-sdk-cpp = aws-sdk-cpp-nix;
       };
 in lib.makeExtensible (self: {
-  nix_2_3 = (common rec {
```

Comment:
NAK.

Please don't do this.

Just bump the package and leave the rest of this alone.

It has been discussed multiple times and many people rely on those.

Reply:
So, I restore from 2.3 to 2.7 ?